Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -132,15 +132,21 @@ void OpenALStream::SoundLoop()
   Common::SetCurrentThreadName("Audio thread - openal");
 
   bool surround_capable = SConfig::GetInstance().bDPL2Decoder;
-#if defined(__APPLE__)
   bool float32_capable = false;
+  bool fixed32_capable = false;
+
+#if defined(__APPLE__)
   const ALenum AL_FORMAT_STEREO_FLOAT32 = 0;
+  const ALenum AL_FORMAT_STEREO32 = 0;
   // OS X does not have the alext AL_FORMAT_51CHN32 yet.
   surround_capable = false;
   const ALenum AL_FORMAT_51CHN32 = 0;
   const ALenum AL_FORMAT_51CHN16 = 0;
+#elif defined(_WIN32)
+  // Only Windows supports AL_FORMAT_STEREO32 alext for now.
+  const ALenum AL_FORMAT_STEREO32 = 0x1203;
```

Comment:
Having varying definitions of this "constant" is confusing. Instead you should probably have a constant variable like `m_format` or whatever, and assign the proper OAL constant to it...